app/models/response.rb

```diff
@@ -4,4 +4,16 @@ class Response < ActiveRecord::Base
 
   validates :poll_id, presence: true
   validates :choice_id, presence: true
+
+  def question
+    self.choice.question.text
+  end
+
+  def text
+    self.choice.text
+  end
+
+  def question_choices
+    self.choice.question.choices
+  end
```

Comment:
We can improve this further by adding another association:
has_many :choices, through: :question
(this uses the other association we just added)
This would makes the response's choices available directly at response.choices.  I think this is what you were looking for.  Sorry for my confusion earlier.